nifi-nar-bundles/nifi-kudu-bundle/nifi-kudu-processors/src/main/java/org/apache/nifi/processors/kudu/PutKudu.java

```diff
@@ -255,12 +272,45 @@ private void trigger(final ProcessContext context, final ProcessSession session,
             final OperationType operationType = OperationType.valueOf(context.getProperty(INSERT_OPERATION).evaluateAttributeExpressions(flowFile).getValue());
             final Boolean ignoreNull = Boolean.valueOf(context.getProperty(IGNORE_NULL).evaluateAttributeExpressions(flowFile).getValue());
             final Boolean lowercaseFields = Boolean.valueOf(context.getProperty(LOWERCASE_FIELD_NAMES).evaluateAttributeExpressions(flowFile).getValue());
+            final Boolean handleSchemaDrift = Boolean.valueOf(context.getProperty(HANDLE_SCHEMA_DRIFT).evaluateAttributeExpressions(flowFile).getValue());
```

Comment:
Same comment applies here. And if you do add a specific attribute to be evaluated against (something like ${handle.schema.drift}), you'll need to check if the attribute does exist in the flow file, otherwise you would have a NPE here.